ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -45,46 +27,132 @@ public AffectedByBugException(String bugNumber) {
    */
   public static void scrub(Map<String, String> attributes, ObjectNode json)
       throws MessageShouldBeDroppedException, AffectedByBugException {
-    // https://bugzilla.mozilla.org/show_bug.cgi?id=1567596
-    if (ParseUri.TELEMETRY.equals(attributes.get(Attribute.DOCUMENT_NAMESPACE))
-        && "crash".equals(attributes.get(Attribute.DOCUMENT_TYPE))
-        && "nightly".equals(attributes.get(Attribute.APP_UPDATE_CHANNEL))
-        && "20190719094503".equals(attributes.get(Attribute.APP_BUILD_ID)) //
+
+    String namespace = attributes.get(Attribute.DOCUMENT_NAMESPACE);
+    String docType = attributes.get(Attribute.DOCUMENT_TYPE);
+    String appVersion = attributes.get(Attribute.APP_VERSION);
+    String appUpdateChannel = attributes.get(Attribute.APP_UPDATE_CHANNEL);
+    String appBuildId = attributes.get(Attribute.APP_BUILD_ID);
```

Comment:
We factor out these repeated attributes to make the if conditions more compact. We also reorganize the cases to group them together by exception type that they throw.

Reply:
This is definitely quite handy